src/model/documents/article.ts

```diff
@@ -0,0 +1,13 @@
+import { BaseModel } from '../base-model';
+import { Category } from './category';
+import { Tax } from './tax';
+
+export interface Article extends BaseModel {
+  imagePath: string;
```

Comment:
What do you think about optional prop `bgColor`? If the user does not provide `imagePath` we will setup `bgColor` to red for example as a default background.